metadata.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: "Metadata"
+description: "Configure and use metadata across the platform"
+---
+
+Metadata are a key data source for understanding and analyzing a workspace's customer feedback.
+
+There are 3 types of metadata we store and utilize in Monterey:
```

Comment:
the word "type" seems overloaded since each metadata key also has a type

what do you think about calling the things that have metadata "Objects" or "Entities" -- some comments below refer to this and say objects, but can align on a term.

Reply:
Changed "type" to "class" in terms of the 3 feedback/customer/account, so now would be `There are 3 classes of metadata we store and utilize in Monterey` (and same change applied elsewhere)